Orange/widgets/evaluate/owpredictions.py

```diff
@@ -299,23 +300,23 @@ def _update_info(self):
             self.info.set_input_summary(self.info.NoInput)
             return
 
-        summary = str(len(self.data)) if self.data else "0"
+        summary = len(self.data) if self.data else "0"
         details = self._get_details()
         self.info.set_input_summary(summary, details)
 
     def _get_details(self):
+        details = format_summary_details(self.data) if self.data else \
+            "No data on input"
+        pred_names = [v.name for v in self.predictors.values()]
         n_predictors = len(self.predictors)
-        if self.data:
-            details = plural("{number} instance{s}", len(self.data))
-        else:
-            details = "No data"
         if n_predictors:
             n_valid = len(self._non_errored_predictors())
-            details += plural("\n{number} model{s}", n_predictors)
+            details += plural("\nModel{s}: {number}", n_predictors)
```

Comment:
Could you split data and model info with a horizontal line. Now it looks like the model is part of the data.
I recommend something like:
```
Data:
303 instances, 14 variables
Features: 13 (7 categorical, 6 numeric)
Target: categorical
Metas: —
---------------------------------------
Models:
2 models (Random Forest, Logistic Regression)
```
If there are no models, use `No models on input` to be consistent with the data info.